source/documentation/administration_guide/topics/proc-adding-kubevirt-openshift-as-an-external-provider.adoc

```diff
@@ -36,7 +36,10 @@ NOTE: This capability is known as _OpenShift Virtualization_.
 
 . Click the *Hosts* tab to verify that the {container-platform} status of the worker nodes is up.
```

Comment:
I know this is not part of your changes but this sentence looks weird. I think it should be this:

Click the *Hosts* tab to verify that the status of the {container-platform} worker nodes is `up`.